test/functional/change_stream_spec_tests.js

```diff
@@ -17,10 +17,12 @@ describe('Change Stream Spec', function() {
 
   before(function() {
     const configuration = this.configuration;
-    return setupDatabase(configuration).then(() => {
-      globalClient = configuration.newClient();
-      return globalClient.connect();
-    });
+    return setupDatabase(configuration)
+      .catch(() => {})
```

Comment:
what is this catching? This smells funky 😄

Reply:
Catching this guy failing to drop the db b/c it does not exist: https://github.com/mongodb/node-mongodb-native/blob/master/test/functional/shared.js#L36

Reply:
should we instead add the catch there, and check for the exact message we hope to skip? I'm afraid this will magically swallow any potential error, and land us in some hot water.

Reply:
pushed new changes